services/QuillLMS/client/app/bundles/PremiumHub/containers/PremiumHub.jsx

```diff
@@ -20,6 +22,9 @@ const PremiumHub = ({ adminId, accessType, passedModel, }) => {
   const [error, setError] = React.useState(null)
   const [showSnackbar, setShowSnackbar] = React.useState(false);
   const [snackbarText, setSnackbarText] = React.useState('');
+  const [hasAppSetting, setHasAppSetting] = React.useState(false);
```

Comment:
Part of me thinks this should be a more verbose variable name in case we ever added a second orthogonal AppSetting to this page, but I can't imagine it will actually matter in practical terms.

Reply:
I think this is a good idea even if there's only one case.